src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -483,12 +485,12 @@ protected virtual ValueTask<bool> HandleAuthorizeWebSocketConnectionAsync(HttpCo
     /// In this manner, both scoped and singleton <see cref="IUserContextBuilder"/>
     /// instances are supported, although singleton instances are recommended.
     /// </summary>
-    protected virtual async ValueTask<IDictionary<string, object?>> BuildUserContextAsync(HttpContext context)
+    protected virtual async ValueTask<IDictionary<string, object?>> BuildUserContextAsync(HttpContext context, object? payload)
```

Comment:
🤔 Is `payload` always `null` here?

Reply:
No. The payload comes in from the WebSocket handler into this protected method so as to make it easy to override this method to change behavior. The WebSocket code hasn’t been merged yet of course. It’s probably a couple thousand lines of code so I’m trying to split it up into smaller pieces for review purposes. It just makes it a little more difficult to see the big picture.

Reply:
Ok